RELEASES.md

```diff
@@ -0,0 +1,112 @@
+# APM Server release checklist
+
+## Before the release
+
+* Backport all relevant commits
+
+* Create a PR that:
+    1. Uncomments out relevant release notes. Release notes are in the [changelogs](https://github.com/elastic/apm-server/tree/master/changelogs).
+    2. Updates the `:stack-version:`
+    * [Sample PR](https://github.com/elastic/apm-server/pull/2064/files)
+
+* Create a separate PR to update `vendor/github.com/elastic/beats/libbeat/version/version.go`
+    * [Sample PR](https://github.com/elastic/apm-server/pull/1886)
+
+* The following may also need to be updated manually:
+    * APM Overview's [release highlights](https://github.com/elastic/apm-server/blob/master/docs/guide/apm-release-notes.asciidoc) - Anything exciting across the APM stack!
+    * APM Overview's [breaking changes](https://github.com/elastic/apm-server/blob/master/docs/guide/apm-breaking-changes.asciidoc) - Any breaking changes across the APM stack.
+    * APM Server's [breaking changes](https://github.com/elastic/apm-server/blob/master/docs/breaking-changes.asciidoc) - Any APM Server breaking changes.
+    * APM Server's [upgrade guide](https://github.com/elastic/apm-server/blob/master/docs/upgrading.asciidoc).
+
+* Changelogs:
+    * Review the [changelogs](https://github.com/elastic/apm-server/tree/master/changelogs) to ensure all relevant notes have been added
+    * Run the [`check_changelogs.py`](script/check_changelogs.py) script to ensure changelogs are synced across branches. This will soon be a PR check.
+        * Don't forget to update the "VERSIONS" to include a new branch if necessary.
+
+## On release day
+
+* New branches need to be added to `conf.yml` in the `elastic/docs` repo. [Example](https://github.com/elastic/docs/pull/893/files#diff-4a701a5adb4359c6abf9b8e1cb38819fR925). **This is handled by the docs release manager.**
+
+* Merge the above PRs
+
+* Push [tags](https://github.com/elastic/apm-server/releases) on GitHub.
```

Comment:
this is now done automatically, let's convert this to a verification step instead